### ISSUE TITLE
Encoders support: announce connection status until the focused encoder connects

### ISSUE THREAD

**Issue by josephsl** [josephsl/stationPlaylist#122]
Hi,

Building from #59 
Would it be possible for nVDA to announce connection attempt messages until the focused encoder connects?

Technical: yes with a new flag, which should not be confused with background monitor flag (Control+F11 from encoders). The benefit of this approach is that people will be able to see if an encoder is really connected rather than NVDA giving up after an error occurs, and is also useful for auto-connect encoders. Note that not all encoders should be treated this way, thus introduce a new seting to preserve old behavior - give up when an error occurs, particularly for archive encoders (reported by at least one broadcaster).

Thanks.
